src/events/mod.rs

```diff
@@ -1441,4 +1497,26 @@ mod test {
         assert_eq!(b.len(), 4);
         assert_eq!(b.name(), QName(b"test"));
     }
+
+    #[test]
+    fn cdata_escaping() {
+        let content = "";
+        let cdata = BytesCData::escaped(content).collect::<Vec<_>>();
+        assert_eq!(cdata.len(), 1);
+        assert_eq!(cdata[0].as_ref(), b"");
+
+        let content = "Certain tokens like ]]> can be difficult and <invalid>";
+        let cdata = BytesCData::escaped(content).collect::<Vec<_>>();
+        assert_eq!(cdata.len(), 2);
+        assert_eq!(cdata[0].as_ref(), b"Certain tokens like ]]");
+        assert_eq!(cdata[1].as_ref(), b"> can be difficult and <invalid>");
+
+        let content = "foo]]>bar]]>baz]]>quux";
+        let cdata = BytesCData::escaped(content).collect::<Vec<_>>();
+        assert_eq!(cdata.len(), 4);
+        assert_eq!(cdata[0].as_ref(), b"foo]]");
+        assert_eq!(cdata[1].as_ref(), b">bar]]");
+        assert_eq!(cdata[2].as_ref(), b">baz]]");
+        assert_eq!(cdata[3].as_ref(), b">quux");
+    }
```

Comment:
Move this test to doc-test of `escaped` method. It is enough to have only doc tests in such simple cases as writing.